Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3 
```

Comment:
might want to change this to `3-onbuild`

Reply:
`onbuild` are deprecated